esx_service/utils/auth_api.py

```diff
@@ -491,6 +491,10 @@ def vm_in_any_tenant(vms):
 def _tenant_vm_add(name, vm_list):
     """ API to add vms for a tenant """
     logging.debug("_tenant_vm_add: name=%s vm_list=%s", name, vm_list)
+    if  name  == auth_data_const.DEFAULT_TENANT:
```

Comment:
Nit: remove the redundant spaces before and after "name". Same for other places.